isis/src/base/apps/isisimport/isisimport.cpp

```diff
@@ -272,6 +272,12 @@ namespace Isis {
     importer.SetBase(base);
     importer.SetMultiplier(multiplier);
 
+    // Update TargetName if Target parameter entered
+    if (ui.WasEntered("TARGET")) {
+      PvlGroup &inst = newLabel.findGroup("Instrument",Pvl::Traverse);
+      inst["TargetName"] = ui.GetString("TARGET");
+    }
```

Comment:
@scsides I decided to implement the user specified target parameter this way instead of trying to add it to the json. Adding it to the json would require updating all existing templates to check for this new keyword and relying on all future templates to include this check as well. Just overriding the existing label seems like a safer way to ensure that the user specified target is adequately updated to the cube label. This also reduces the checks needed to be added to each template.

Reply:
Good reason. We might be able to put something down in the parent template(s) to avoid the duplication.